WalletWasabi.Fluent/Views/CoinControl/Core/Cells/IndicatorsCellView.axaml

```diff
@@ -0,0 +1,52 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:cells="clr-namespace:WalletWasabi.Fluent.ViewModels.CoinControl.Core.Cells"
+             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+             x:DataType="cells:IndicatorsCellViewModel"
+             x:CompileBindings="True"
+             x:Class="WalletWasabi.Fluent.Views.CoinControl.Core.Cells.IndicatorsCellView">
```

Comment:
Can we keep same naming scheme for templates, everywhere else we use `Columns`, in this PR we use cells (both views and view models).

Reply:
I think we're messing things up in other places, because this view is used to render cells of a column, not the column itself.

In fact, the parameter name of template column is "cellTemplate".

![image](https://user-images.githubusercontent.com/3109851/198991941-a0dbccf3-4597-4541-a847-c766cfbf6776.png)